kura/org.eclipse.kura.web2.jetty/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: org.eclipse.kura.web2.jetty
```

Comment:
I would name it as org.eclipse.kura.jetty.customizer since the fragment provides a customisation and attaches to jetty and not the web 2 bundle that sits on top of the jetty web server. What do you think?

I also would move this to the target platform folder.